parser/v2/types.go

```diff
@@ -194,6 +194,7 @@ func (exp TemplateFileGoExpression) Write(w io.Writer, indent int) error {
 	return err
 }
 
+//nolint:unused // This could be useful.
```

Comment:
What's the deal with this? Not sure why we'd need an unused variable.

Reply:
It can be removed, I figured it could be a useful function in the future, but I suppose we could grab it from git history.